### DISCUSSION TITLE
Deye SUN-6K-SG05LP1-EU does not work.

### DISCUSSION THREAD

**Discussion by kodr5555** [StephanJoubert/home_assistant_solarman#600] [Q&A]
I tried all the DEYE configurations, none of them worked! Model: SUN-6K-SG05LP1-EU Logger: 27ХХХХХХХХ Logger Firmware: LSW3_15_FFFF_1.0.9E.

**Comment by PV-Joe** [comment]
I’m using the Solarman Integration 1.4.3 with my Deye Hybrid 8K without any Problems.
I never updated to the latest version.

**Reply by kodr5555** [reply]
I tried installing version 1.4.3 but it didn’t work

**Reply by PV-Joe** [reply]
If you tried a Version higher than 1.4.3 you have to delete all remaking files manually first an restart before downgrading to 1.4.3.

**Reply by kodr5555** [reply]
Thank you this helped!!!

**Comment by felmian** [comment]
I have the same problem. device serial 29xxxx Firmware versionLSW3_15_FFFF_1.0.9E

**Reply by felmian** [reply]
Less time passed and everything worked
